www/index.html

```diff
@@ -38,23 +20,154 @@
+            </div>
+            <div class="panel-body">
+                <ol>
+                    {{ #stops }}
+                        <li>{{ name }}</li>
+                    {{ /stops }}
+                </ol>
+            </div>
+        </div>
+    </script>
+
+    <script id="departures-tmpl" type="x-tmpl-mustache" >
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h3 class="panel-title">Dia de Semana</h3>
+            </div>
+            <div class="panel-body">
+                <ul class="list-inline unstyled">
+                    {{ #WEEKDAY }}
+                        <li>{{ time }}</li>
+                    {{ /WEEKDAY }}
+                </ul>
+            </div>
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h3 class="panel-title">Sábado</h3>
+            </div>
+            <div class="panel-body">
+                <ul class="list-inline unstyled">
+                    {{ #SATURDAY }}
+                        <li>{{ time }}</li>
+                    {{ /SATURDAY }}
+                </ul>
+            </div>
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h3 class="panel-title">Domingo</h3>
+            </div>
+            <div class="panel-body">
+                <ul class="list-inline unstyled">
+                    {{ #SUNDAY }}
+                        <li>{{ time }}</li>
+                    {{ /SUNDAY }}
+                </ul>
+            </div>
+        </div>
+    </script>
+
+    <script id="routes-result-tmpl" type="x-tmpl-mustache">
+        {{ #empty }}
+            <div class="alert alert-warning alert-dismissible fade in" role="alert">
+                <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+                <strong>Oops!</strong> Nenhum registro encontrado.
+            </div>
+        {{ /empty }}
+
+        {{ ^empty }}
+            <div class="table-responsive">
+                <table class="table table-hover table-striped">
+                    <thead>
+                        <tr>
+                            <th>#</th><th>Nome</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {{ #routes }}
+                        <tr data-route-id="{{ id }}" data-route-name="{{ longName }}">
+                            <td>{{ shortName }}</td>
+                            <td>{{ longName }}</td>
+                        </tr>
+                        {{ /routes }}
+                    </tbody>
+                </table>
+            </div>
+        {{ /empty }}
+    </script>
+
+
+    <script type="text/javascript" src="cordova.js"></script>
+    <!-- external dependencies -->
+    <script src="./libs/jquery/3.1.1/jquery.min.js"></script>
```

Comment:
Yeah, I agree that a build tool would be great. 
However my experience using JS' build tools like grunt and gulp (never used webpack) is limited to executing commands to generated the assets my BE code would need. I don't really have much experience configuring builds on them. 
Thus, despite recognizing the advantage of using one to concatenate file to diminish the amount of HTTP requests and diminish the load time as well as generating versioned files to avoid caching issues (while I'm actually using versioned files for external libraries), I have decided not to use them.